test/Foo-test.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
```

Comment:
I should remove this file, as i was using it to test if enzyme was working correctly.